app/backend/src/couchers/servicers/api.py

```diff
@@ -120,6 +122,26 @@
     ParkingDetails.paid_offsite: api_pb2.PARKING_DETAILS_PAID_OFFSITE,
 }
 
+fluency2sql = {
+    api_pb2.LanguageAbility.Fluency.FLUENCY_UNSPECIFIED: None,
+    api_pb2.LanguageAbility.Fluency.FLUENCY_SAY_HELLO: LanguageFluency.say_hello,
+    api_pb2.LanguageAbility.Fluency.FLUENCY_BEGINNER: LanguageFluency.beginner,
+    api_pb2.LanguageAbility.Fluency.FLUENCY_INTERMEDIATE: LanguageFluency.intermediate,
+    api_pb2.LanguageAbility.Fluency.FLUENCY_ADVANCED: LanguageFluency.advanced,
+    api_pb2.LanguageAbility.Fluency.FLUENCY_FLUENT: LanguageFluency.fluent,
+    api_pb2.LanguageAbility.Fluency.FLUENCY_NATIVE: LanguageFluency.native,
+}
+
+fluency2api = {
+    None: api_pb2.LanguageAbility.Fluency.FLUENCY_UNSPECIFIED,
```

Comment:
Remove from conversion dict; the database will never return None because of the `nullable=False` column property.